### PR TITLE
Update missing paths from go/wbb-oc-paths to BGP feature pro…

### PR DESCRIPTION
**Pull request by hariramnat**
…files

### CODE REVIEW

File: feature/bgp/feature.textproto [hunk 1]
```diff
@@ -33,6 +33,23 @@ telemetry_path {
   path: "/network-instances/network-instance/protocols/protocol/bgp/global/state/router-id"
 }
 
+# Notification error parameters
```
Comment (dplore) [inline feature/bgp/feature.textproto:36]: could combine into # messages section below?

File: feature/bgp/feature.textproto [hunk 2]
```diff
@@ -55,6 +72,25 @@ telemetry_path {
 telemetry_path {
   path: "/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/state/session-state"
 }
+telemetry_path {
+  path: "/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/state/active"
+}
+
```
Comment (dplore) [inline feature/bgp/feature.textproto:78]: no space here, just to keep consistent with existing style?

File: feature/bgp/policybase/feature.textproto [hunk 3]
```diff
@@ -272,6 +272,25 @@ telemetry_path {
 telemetry_path {
   path: "/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/state/prefixes/sent"
 }
+telemetry_path {
+  path: "/network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/afi-safis/afi-safi/apply-policy/state/export-policy"
+}
+telemetry_path {
+  path: "/network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/afi-safis/afi-safi/apply-policy/state/import-policy"
+}
+
```
Comment (dplore) [inline feature/bgp/policybase/feature.textproto:281]: no space here, just to keep consistent with existing style

File: feature/bgp/prefixlimit/feature.textproto [hunk 4]
```diff
@@ -32,3 +32,18 @@ config_path {
...
+}
+telemetry_path {
+  path: "/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/ipv6-unicast/prefix-limit/state/restart-timer"
+}
+telemetry_path {
+  path: "/network-instances/network-instance/protocols/protocol/bgp/neighbors/neighbor/afi-safis/afi-safi/ipv6-unicast/prefix-limit/state/warning-threshold-pct"
+}
+{
+  path: "/network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/afi-safis/afi-safi/ipv6-unicast/prefix-limit/state/warning-threshold-pct"
+}
```
Comment (dplore) [inline feature/bgp/prefixlimit/feature.textproto:46]:
```suggestion
telemetry_path {
  path: "/network-instances/network-instance/protocols/protocol/bgp/peer-groups/peer-group/afi-safis/afi-safi/ipv6-unicast/prefix-limit/state/warning-threshold-pct"
}
```

File: feature/isis/feature.textproto [hunk 5]
```diff
@@ -134,3 +148,16 @@ config_path {
 telemetry_path {
   path: "/network-instances/network-instance/protocols/protocol/isis/interfaces/interface/afi-safi/af/state/enabled"
 }
+
```
Comment (dplore) [inline feature/isis/feature.textproto:151]: no space

**Review by dplore — CHANGES_REQUESTED** [review]
few minor nits and one syntax issue

File: feature/bgp/feature.textproto (hunk 1 above)
Comment (hariramnat) [reply feature/bgp/feature.textproto]: Done!

File: feature/bgp/feature.textproto (hunk 2 above)
Comment (hariramnat) [reply feature/bgp/feature.textproto]: done!

File: feature/bgp/prefixlimit/feature.textproto (hunk 4 above)
Comment (hariramnat) [reply feature/bgp/prefixlimit/feature.textproto]: Done!

File: feature/isis/feature.textproto (hunk 5 above)
Comment (hariramnat) [reply feature/isis/feature.textproto]: Done!

**Review by dplore — APPROVED** [review]

**Pull request merged**